### ISSUE TITLE
Negative TC in loss computation

### ISSUE THREAD

**Issue by Greek-Guardian** [zpham/D3VAE#1]
In the file [https://github.com/zpham/D3VAE/blob/main/exp/exp_model.py](url)
`loss = loss1*self.args.psi + loss2*self.args.lambda1 + mse_loss - self.args.gamma*total_c`
Why is total_c negative while we want it be small?
